Constellation/Node/Main.hs

```diff
@@ -86,7 +87,10 @@ run cfg@Config{..} = do
             }
     ast <- mustLoadPublicKeys cfgAlwaysSendTo
     logf' "Initializing storage {}" [cfgStorage]
-    storage <- berkeleyDbStorage cfgStorage
+    storage <- case break (== ':') cfgStorage of
+        ("bdb", ':':s) -> berkeleyDbStorage s
+        ("dir", ':':s) -> directoryStorage s
+        _              -> berkeleyDbStorage cfgStorage  -- Default
     -- storage <- memoryStorage
```

Comment:
I think it's worth having for now, until a final decision is made wrt storage to use - that way you don't have to have the commented out tests and dependencies in the code.